### DISCUSSION TITLE
Redundant Artifacts

### DISCUSSION THREAD

**Discussion by wkirgsn** [moetomg/magnet-engine#3] [General]
Hi,
I think the models .pt files and the Models code is redundant now with the mag-net-hub lib and can be removed.
That is, everything under /src/teams, right?

**Comment by moetomg** [comment]
Hi, Wilhelm

Yeah, you are right. I have already merged the pip package into the main program, thus the model files are redundant.

Best regards,
Qiujie

On Jun 29, 2024, at 22:18, Wilhelm Kirchgässner ***@***.***> wrote:

﻿

Hi,
I think the models .pt files and the Models code is redundant now with the mag-net-hub lib and can be removed.
That is, everything under /src/teams, right?
